js/components/graph/Sidebar.js

```diff
@@ -1,173 +1,72 @@
-      // open graph
-      this.setState({
-        toggled: true,
-        contentHidden: false,
-        graphActive: 1,
-      });
-    }
-  }
-
-  showFocuses = focus => {
-    if (focus) {
-      // show focuses
-      this.setState({
-        graphActive: 0
-      });
-    } else {
-      // show graphs
-      this.setState({
-        graphActive: 1
-      });
-    }
+  toggleSidebar = () => {
+      this.setState({ hidden: !this.state.hidden });
   }
 
   // Sidebar rendering methods
-  renderSidebarHeader= () => {
-    const contentHiddenClass = this.state.contentHidden ? "hidden" : "";
+  /**
+   * Render the FCE counter above the sidebar on the left side.
+   * @return {HTMLDivElement} FCE to the DOM
+   */
+  renderFCE = () => {
     const fceString = Number.isInteger(this.props.fceCount) ? this.props.fceCount + ".0" : this.props.fceCount
 
     return (
-      <div id="fce" className={contentHiddenClass}>
-        <div id="fcecount" data-testid="test-fcecount">FCE Count: {fceString}</div>
-        <button id="reset" data-testid="test-reset" onClick={() => this.props.reset()}>Reset Selection</button>
-      </div>
-    )
-  }
-
-  renderSidebarNav = () => {
-    const focusDisabled = this.state.focusDisabled ? "disabled" : "";
-    const focusActiveClass = this.state.graphActive === 0 ? "active" : "";
-
-    return (
-      <nav id="sidebar-nav">
-        <ul>
-
-          <li
-            id="focuses-nav"
-            className={`${focusActiveClass} ${focusDisabled}`}
-            onClick={() => this.showFocuses(true)}
-            data-testid="test-focuses-nav"
-          >
-            <div className="nav-section">Focuses</div>
-          </li>
-        </ul>
-      </nav>
+      <div className="fcecount" data-testid="test-fcecount">FCE Count: {fceString}</div>
     )
   }
 
-  renderSidebarButtons = () => {
-    const focusHiddenClass = this.state.graphActive === 1 ? "hidden" : "";
-
+  /**
+   * Render courses that are in the sidebar.
+   * @return {HTMLBodyElement} list of div's for each course that is active
+   */
+  renderActiveCourses = () => {
+    let temp = this.props.activeCourses ? [...this.props.activeCourses] : [];
+    // sort the list of rendered courses, alphabetically
+    temp.sort((a,b) => a.localeCompare(b));
     return (
-      <div>
-        <div id="focuses" className={focusHiddenClass} data-testid="test-focus-buttons">
-          {this.createFocusButtons()}
-        </div>
+      <div className="courses" data-testid="test-course-selection">
+        {temp.map((course) => {
+          return (
+            <div key={`active ${course}`} data-testid={`test ${course}`} className="course-selection">{course.toUpperCase()}</div>
+          );
+        })}
       </div>
-    )
+    );
   }
 
   render() {
-    const flippedClass = this.state.toggled ? "flip" : "";
-    const sidebarClass = this.state.toggled ? "opened" : "";
+    const collapsedClass = this.state.hidden ? "collapsed" : "expanded";
```

Comment:
Your change is okay, but missed the actual point of my previous comment: the **state** `hidden` should be renamed.